_sass/_custom-components.scss

```diff
@@ -38,3 +38,15 @@
     }
   }
 }
+
+// ==========================================================================
+// Meta info - Seen on News & Updates and Block in Content samples
+// ==========================================================================
+.meta-info.meta-info--block {
```

Comment:
8. Custom styles for the _Last Updated_ box on the Content page.

---

### We've reached the end 🥳

To summarize we've:
1. Updated Header and Megamenu colors
1. Updated branding and styling Footer and Identifier
1. Added sample markup and theming for the _last updated_ box on Content pages